nibabel/freesurfer/tests/test_io.py

```diff
@@ -92,6 +92,11 @@ def test_morph_data():
     curv = read_morph_data(curv_path)
     assert_true(-1.0 < curv.min() < 0)
     assert_true(0 < curv.max() < 1.0)
+    with InTemporaryDirectory():
+        new_path = 'test'
+        write_morph_data(new_path, curv)
+        curv2 = read_morph_data(new_path)
+        assert_equal(curv2, curv)
 
```

Comment:
Maybe a test reading an existing curv data file from freesurfer examples, re-writing and reading again?  E.g. `nibabel-data/nitest-freesurfer/fsaverage/surf/lh.curv` using decorator like : https://github.com/nipy/nibabel/blob/master/nibabel/freesurfer/tests/test_io.py#L48